nuxeo-coldstorage-web/rollup.config.js

```diff
@@ -8,43 +8,46 @@ import path from 'path';
 // https://github.com/nuxeo/nuxeo-web-ui/blob/master/index.js#L7
 const GLOBALS = {
   '@polymer/polymer/lib/legacy/class.js': 'const { mixinBehaviors } = Polymer; export { mixinBehaviors };',
-  '@polymer/polymer/lib/legacy/polymer.dom.js': 'const { dom, matchesSelector } = Polymer; export { dom, matchesSelector };',
-  '@polymer/polymer/polymer-element.js': 'const { PolymerElement} = window; export { PolymerElement };',
-  '@polymer/polymer/lib/elements/dom-module.js': 'const { DomModule} = window; export { DomModule };',
-  '@polymer/polymer/lib/legacy/polymer.dom.js': 'const { dom, html, matchesSelector } = Polymer; export { dom, html, atchesSelector };',
+  '@polymer/polymer/lib/legacy/polymer.dom.js':
+    'const { dom, html, matchesSelector } = Polymer; export { dom, html, matchesSelector };',
```

Comment:
extra space at the beginning of line `12`

Reply:
Agree that it may seem odd, but it was automatically formatted by prettier. 😅 🤷🏼‍♂️